README.md

```diff
@@ -1 +1,43 @@
-# node-arguments-lab
\ No newline at end of file
+# Node Arguments Lab
+
+In this lab, you will pass and access command-line arguments when working with Node.
+
+## Objectives
+
+1. Implement a Node script that takes CLI arguments and prints them back
+1. Use two arguments
+
+## Introduction
+
+Knowing how to pass and access command-line arguments is important in building CLI tools or small programs like the program which gives a random password.
```

Comment:
Make the intro more project based - check out [this example](https://github.com/learn-co-curriculum/js-anonymous-functions-lab/blob/wp/README.md) for a refernce